build.xml

```diff
@@ -62,7 +62,6 @@
     <include name="apache_commons/commons-io-2.4.jar"/>
     <include name="apache_commons/commons-lang3-3.1.jar"/>
     <include name="mongo/mongo-java-driver-3.8.2.jar"/>
-    <include name="bson4jackson/bson4jackson-2.2.0-2.2.0.jar"/>
```

Comment:
Did this sneak in from another cause/change? (That's my guess)

Reply:
dependency tree cruft that was missed when removing the root jar in the tree

Reply:
Really need to move to gradle at some point

Reply:
1. If you get a chance, splitting this change out into its own commit helps.

2. What's the next step re: Gradle: create a ticket?

Reply:
For 1): yeaaaah... I'm not going to spend my time on that. Sorry.
For 2): Nothing, I'll get to it when I get to it. It's not something that needs to be tracked.